t/lib/OpenQA/Test/Utils.pm

```diff
@@ -467,13 +465,14 @@ sub unstable_worker {
             });
         setup_worker($worker, $host);
         $worker->init();
+        Devel::Cover::report() if Devel::Cover->can('report');
         if ($ticks < 0) {
+            # This call will effectively never return
             Mojo::IOLoop->singleton->start;
```

Comment:
maybe instead of creating a subprocess and then calling `Mojo::IOLoop::start` we could use the functionality from `Mojo::IOLoop` directly and do something like `Mojo::IOLoop::subprocess`

Reply:
I want for a ridiculous amount of ticks, which means we can use the same code path